.github/workflows/create-release.yml

```diff
@@ -41,8 +41,21 @@ jobs:
           token: ${{ secrets.RELEASE_TOKEN }}
           # Ensure we are checked out on the develop branch
           ref: develop
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
       - name: Bump version
-        run: node ./scripts/release/bump-version.js "${{ needs.release-pr.outputs.new-version }}"
+        run: npm version "${{ needs.release-pr.outputs.new-version }}"
```

Comment:
Different from the `bump-version.js`, the `npm version` will make a commit and create a new version tag.

We need to add `--no-commit-hooks` and `--no-git-tag-version` to skip these two actions.

```sh
npm version --no-commit-hooks --no-git-tag-version "${{ needs.release-pr.outputs.new-version }}"
```